src/fileops.rs

```diff
@@ -6,27 +6,30 @@ use std::ffi::CString;
 use libc::c_int;
 use std::os::unix::io::{RawFd, AsRawFd};
 use std::ops::Drop;
+use std::io::Error;
+use std::error::Error as std_error;
```

Comment:
Thanks rust for having multiple Error s that i need to import.... at least i dont have to use error::Error s... yet(?)